src/components/Rockets.js

```diff
@@ -1,9 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRocketsData } from '../redux/rockets/rocketsSlice';
 
-const Rockets = () => (
-  <div>
-    <h2>Rockets</h2>
-  </div>
-);
+const Rockets = () => {
+  const rocketsData = useSelector((state) => state.rocketsList);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRocketsData());
+  }, [dispatch]);
+
+  if (rocketsData.loading) return <p className="fetchStatus">Loading...</p>;
+  if (rocketsData.errors) return <p className="error">Error loading the rockets</p>;
+  return (
+    <ul className="rocketsList">
+      {rocketsData.value.map((e, i) => (
+        <>
+          key=
+          {e.id}
+          id=
+          {e.id}
+          indexNo=
+          {i}
+          title=
+          {e.name}
+          description=
+          {e.description}
+          img=
+          {e.flickrImages[0]}
+          reserved=
+          {e.reserved}
+        </>
+      ))}
+    </ul>
```

Comment:
Diana,

I noticed that Inside the map function, the code is using React Fragments without wrapping them in any JSX element. Fragments should be wrapped within an element or self-closing tags.

I also noticed that the attributes inside the map function are not properly formatted with JSX syntax.

Lastly, I noticed you are displaying all of the data fetched from the API and you didn't format the fetched data properly to meet the [design requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md).

Here is a way you can implement this

```suggestion
      <ul className="rocketsList">
      {rocketsData.value.map((e, i) => (
        <li key={e.id}>
          {e.flickrImages[0] && <img src={e.flickrImages[0]} alt={e.name} />}
          <h2>{e.name}</h2>
          <p>
            {e.reserved && <span>Reserved</span>} {e.description}
          </p>
        </li>
      ))}
    </ul>
```